type-object/src/test/java/com/iluwatar/type/object/BreedTest.java

```diff
@@ -0,0 +1,33 @@
+package com.iluwatar.type.object;
+
+import static org.junit.Assert.*;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * 
+ * 
+ * @author JAPO
+ *
+ */
+public class BreedTest {
+  
+  /**
+   * 
+   * Test if Monster created by the correct attributes, as requested
+   */
+  @Test
+  public void testBreedBreedTypeIntString() throws Exception {
+    
+    Breed beast = new Breed(BreedType.BEAST, 25, "Beast attack");    
+    Breed beastHunter = new Breed(BreedType.BEAST_HUNTER, beast, 30, null);
+        
+    assertNotNull(BreedType.BEAST);
+    assertNotNull(BreedType.BEAST_HUNTER);
+    assertNotNull(BreedType.BEAST_HUNTER.toString());
+    
+  }
+
+}
```

Comment:
The JavaDoc says we are testing `Monster` instances but that is not valid here. Maybe change this to `MonsterTest` and instantiate `Monster`s and test their attributes?